### PR TITLE
Update xeus-sqlite example notebook

### PR DESCRIPTION
**Pull request by psychemedia**
- annotate examples;
- identify limitations.

### CODE REVIEW

File: content/xeus-sqlite/simple-operations.ipynb [hunk 1]
```diff
@@ -15,22 +15,42 @@
...
+      "metadata": {}
+    },
     {
       "cell_type": "code",
       "source": "CREATE TABLE players (Name STRING, Class STRING, Level INTEGER, Hitpoints INTEGER)",
       "metadata": {
         "trusted": true
       },
-      "execution_count": null,
+      "execution_count": 2,
```
Comment (jtpio) [inline content/xeus-sqlite/simple-operations.ipynb:53]: Let's keep the notebook without execution counts and output?

Comment (psychemedia) [reply content/xeus-sqlite/simple-operations.ipynb:53]: For someone wanting a quick preview of what's possible, showing run cells means folk can just preview the notebooks with outputs without having to run them?

It also shows what's expected if things *don't* work for whatever reason?

But that's my personal preference...

Comment (jtpio) [reply content/xeus-sqlite/simple-operations.ipynb:53]: ah ok fair enough. Some other notebooks in this repo have cleared outputs so it was mostly to keep things consistent.

**Review by jtpio — APPROVED** [review]
Thanks!

**Comment by psychemedia** [comment]
@jtpio If required/ appropriate, I am happy to revise the other demo notebooks in a similar tone / style?

**Comment by jtpio** [comment]
Maybe it makes sense for notebooks. But for widgets maybe it's better to regenerate live?

**Comment by psychemedia** [comment]
@jtpio Yes, agreed, widget notebooks donlt really do static previews very well unless the widget is completely self-contained  / standalone js app that just happens to be created as cell output, rather than a `ipywidget`, for example, that expects to talk to py state.

**Pull request merged**